vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -393,8 +396,13 @@ void Unblinded::OnProcessTokens(
     redeem.token_list = token_list;
     redeem.contribution_id = contribution->contribution_id;
 
-    credentials_->RedeemTokens(redeem, redeem_callback);
-    return;
+    if (redeem.processor == ledger::ContributionProcessor::UPHOLD ||
```

Comment:
for this two processors we need to go through different credentials flow as for brave funds